modules/post/windows/gather/credentials/plsql_developer.rb

```diff
@@ -0,0 +1,131 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::Windows::UserProfiles
+  include Msf::Post::File
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Windows Gather PL/SQL Developer History and Passwords',
+        'Description' => %q{
+          This module can decrypt the history of PL/SQL Deceloper,
```

Comment:
```suggestion
          This module can decrypt the history of a PL/SQL Developer,
```

Reply:
`PL/SQL Developer` is the name of the software. Not sure about whether `a` is needed here?